integrationtests/mayavi/common.py

```diff
@@ -316,6 +316,9 @@ class TestCase(Mayavi):
     # `self.compare_image` was called in the test..
     offscreen = Bool(False)
 
+    # Profile for memory usage.
+    profile = Bool(False)
```

Comment:
Two main reasons:
(i) just `profile` can mean memory or time profiling
(ii) profiling in general means collecting detailed information during the code run (e.g see https://pypi.python.org/pypi/memory_profiler, python profiler). In our case we are not creating a memory profile of the run over time (or lines or function calls) but we are reporting and checking the memory usage in a much more crude way.